### DISCUSSION TITLE
luma.gl - multiple VERSIONs detected: 8.3.1 vs 8.3.2

### DISCUSSION THREAD

**Discussion by novakand** [visgl/deck.gl#5272] [Q&A]
Good day! I install npm install @ deck.gl / core @ deck.gl / google-maps init.js error occurs: 35 Uncaught Error: luma.gl - Multiple VERSIONS found: 8.3.1 vs 8.3.2.
@ deck.gl package.json 
```
"dependencies": {
    "@ loaders.gl / core": "^ 2.3.0",
    "@ loaders.gl / images": "^ 2.3.0",
    "@ luma.gl / core": "^ 8.3.1",
    "@ math.gl / web-mercator": "^ 3.3.0",
    "gl-matrix": "^ 3.0.0",
    "math.gl": "^ 3.3.0",
    "mjolnir.js": "^ 2.3.0",
    "probe.gl": "^ 3.2.1"
  },
```

**Comment by Pessimistress** [comment]
You can run `npm ls @luma.gl/core` or `yarn list @luma.gl/core` to check your installed packages. You must have only one version installed.

This is usually caused by your local lock files.

**Comment by davidcalhoun** [comment]
I'm getting a similar error when installing with Yarn - `Error: luma.gl - multiple VERSIONs detected: 8.5.14 vs 8.5.10`

With newer yarn you can use this command: `yarn why @luma.gl/core`

If you see multiple versions of luma here, you can use this workaround to lock it down to one specific version.  Add this to your `package.json`:

```
    "resolutions": {
        "@luma.gl/core": "8.5.10"
    }
```

Then run `yarn`.  If you still see the error after this, you may need to run `yarn cache clean` and `yarn` again to reinstall.

**Reply by davidcalhoun** [reply]
Getting this again when updating deck.gl from `8.7.11` to `8.7.12`: `Error: luma.gl - multiple VERSIONs detected: 8.5.12 vs 8.5.14`.  Locked down `@luma.gl/@core` to `8.5.14` this time and that seemed to fix the issue.
